### DISCUSSION TITLE
快照捕获不到“自动精灵”应用的“选择运行模式”弹窗

### DISCUSSION THREAD

**Discussion by Tsuk1ko** [gkd-kit/gkd#800] [Q&A] [answered]
https://i.gkd.li/i/17968731

想通过 gkd 自动去确认[自动精灵](http://zdjl.cc/)的一些对话框，但唯独这部分快照无法捕获到，只能捕获后方界面内容，不清楚是自动精灵这里有什么特殊性还是 gkd 的原因

![image](https://github.com/user-attachments/assets/01c4400c-b49e-4b04-b563-0537b3d8b619)

**Comment by lisonge** [comment, marked as answer]
应该是这个对话框没有提供无障碍信息，所以获取不到

你用其他框架获取应该也是一样的结果

**Reply by Tsuk1ko** [reply]
👌感谢回答
